applications/notifications/server/notifications/services/notification/backends/email_backend.py

```diff
@@ -0,0 +1,45 @@
+import smtplib
```

Comment:
With the nomenclature we are agreeing this should be considered a "middleware", not a "backend". But maybe I like the "backend" nomenclature as it does not conflict with the middlewares as seen by redux. @zsinnema what do you think?

Reply:
@filippomc I understand your point and also I understand middleware. I suggest that we start using the term "backend"